docs/src/config.md

```diff
@@ -20,3 +22,51 @@ catch
     @warn "Could not turn on Rebugger key bindings."
 end
 ```
+
+## Customize keybindings
+
+It's possible that Rebugger's default keybindings don't work for you.
+This can be for various reasons. Some window managers override the Rebugger's
+keybindings with their own. Some terminals map the keybindings used by Rebugger
+to different escape sequences than those hardcoded in Rebugger. You can work
+around these issues by adding your own keybindings.
+
+To add your own keybindings, use `Rebugger.add_keybinding(action, keybinding)`.
+This can be done during a running Rebugger session. Here is an example that
+maps the "step in" action to the key "F5":
+```julia
+julia> Rebugger.add_keybinding(action=:stacktrace, keybinding="\e[15~")
+```
+
+To make your keybindings permanent, add them to your `startup.jl` file like so:
+```julia
+try
+    @eval using Revise
+    # Turn on Revise's file-watching behavior
+    Revise.async_steal_repl_backend()
+catch
+    @warn "Could not load Revise."
+end
+
+try
+    @eval using Rebugger
+    # Activate Rebugger's key bindings
+    Rebugger.keybindings[:stepin] = "\e[17~"  # Add the keybinding F6 to step into a function.
+    Rebugger.keybindings[:stacktrace] = "\e[18~"  # Add the keybinding F6 to print a stacktrace.
+    atreplinit(Rebugger.repl_init)
+catch
+    @warn "Could not load Rebugger."
+end
+```
+
+But how to find out the cryptic string that corresponds to the keybinding you
+want? Use Julia's `read()` function:
+
+```julia
+julia> read(stdin, String)
+^[[15~"\e[15~"
+```
+
+After calling `read()`, press the keybinding that you want. Then, press `Ctrl+D`
+twice to terminate the input. `read()` will return the cryptic string you are
+looking for.
```

Comment:
Nice. How about write it as
```julia
julia> str = read(stdin, String)
^[[15~"\e[15~"

julia> str
"\e[15~"
```
and then say "the value of `str` is the cryptic string you are looking for". (The fact that it prints the string immediately after the funny char representation might still make some folks confused about which part of it is relevant.)

Reply:
Makes sense.